tests/data/resource/virtual_codebase/fingerprint_attribute.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
This serves as a potential test case because if we currently create a `VirtualCodebase` with the given file it will throw an exception because it is creating `Resource attributes` only upon looking at the top directory/file here:
https://github.com/nexB/commoncode/blob/a1744eb4f4308598ea166e4ff1025356533f6054/src/commoncode/resource.py#L1596

The test JSON I provided contains an additional attribute `fingerprint` in a file `Resource`.

Reply:
That's great. You trimmed the test JSON and this now becomes very clear.